intel-sgx/enclave-runner/src/loader.rs

```diff
@@ -332,6 +332,7 @@ impl<'a> EnclaveBuilder<'a> {
     }
 
     pub fn build<T: Load>(mut self, loader: &mut T) -> Result<Command, Error> {
+        self.initialized_args_mut();
```

Comment:
Looking at the the source code around, my thinking would be that the `build_library` function needs arguments being set as well, to avoid running a `main` function without arguments at all. Opinions @raoulstrackx @mkaynov ?

Alternatively, we could move the logic of a fallback signle argument to the `load` function. This way, the failsafe code would be at one place, instead of across these individual functions.